ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3618,10 +3618,10 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-shared-to-local-gateway-mode-migration
-- as: vsphere-ipi-ovn-shared-to-local-gw-migration-f28-destructive
-  cron: 36 6 6 * *
+- as: vsphere-ipi-ovn-winc-p2-f14
```

Comment:
It seems there has been some conflict during the merge as your PR is overriding an existing job instead of adding a new one vsphere-ipi-ovn-winc-p2-f14. This job should be left as it is [right now](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml#L3707C1-L3716)